<ui version="4.0" >
 <class>MvdFilterWidget</class>
 <widget class="QFrame" name="MvdFilterWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="frameShape" >
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow" >
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>4</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="closeButton" >
     <property name="toolTip" >
      <string>Clear filter and close filter bar.</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset/>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="filterLabel" >
     <property name="text" >
      <string>Filter:</string>
     </property>
     <property name="buddy" >
      <cstring>input</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MvdClearEdit" name="input" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="whatsThis" >
      <string>Filter movies by showing only those who match the entered criteria. See the application preferences to customize the behavior of the quick filter.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="caseSensitive" >
     <property name="text" >
      <string>Case Sensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="warningIconLabel" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="warningTextLabel" >
     <property name="text" >
      <string>No movies match the filter criteria</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>51</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MvdClearEdit</class>
   <extends>QLineEdit</extends>
   <header>mvdshared/clearedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
